src/pages/api/fscheck.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { Database } from "../../lib/database";
+
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  const fs = new Database();
+  let isConnected = await fs.isConnected();
+  let statusCode = isConnected ? 200 : 503;
```

Comment:
nit: could this possibly return some information about the firestore and if errors were emitted that could get bundled in like:
```
{
   status: 503,
   error: {...} // Emitted error after polling is completed without firestore connection
}
```

Reply:
Hmm, not sure I really see a need for this. This is specifically what the `fscheck` health check checks; either it can connect (`200`) or it can't (`503`). I even considered just doing this as a HEAD request, but figured a GET might be easier for some callers to check. If you think more is needed, please open an issue.

Reply:
Can do 👍🏼